app/routines/exercises/import/assessments.rb

```diff
@@ -69,15 +69,15 @@ def exec(filename:, book_uuid:)
               end
               raise ArgumentError, 'Could not find "Question Stem" column' if question_stem_index.nil?
 
-              uuid_index ||= headers.index do |header|
-                header == 'uuid' || header == 'page uuid' || header == 'section uuid'
+              uuid_index ||= headers.index { |header| header&.end_with?('uuid') && !header.include?('book') }
+              section_index ||= headers.index do |header|
+                header&.start_with?('section') && !header.include?('uuid') && !header.include?('name')
```

Comment:
Does algebra have slightly different headers? Maybe it's not a worry if we are sure about the imported format, but I just want to double check that loosening the matching is not going to be a concern.

Reply:
it was a mess of different headers, let me say that